thoth/storages/graph/postgres.py

```diff
@@ -4727,17 +4726,42 @@ def sync_solver_result(self, document: dict) -> None:
                         metadata_version=importlib_metadata.pop("Metadata-Version", None),
                         # package name
                         name=importlib_metadata.pop("Name", None),
-                        platform=importlib_metadata.pop("Platform", None),
-                        requires_dist=importlib_metadata.pop("Requires-Dist", None),
                         summary=importlib_metadata.pop("Summary", None),
                         # package version
                         version=importlib_metadata.pop("Version", None),
                         requires_python=importlib_metadata.pop("Requires-Python", None),
-                        description_content_type=importlib_metadata.pop("Description-Content-Type", None),
-                        project_url=importlib_metadata.pop("Project-URL", None),
-                        provides_extra=importlib_metadata.pop("Provides-Extra", None),
+                        description=importlib_metadata.pop("Description", None),
+                        description_content_type=importlib_metadata.pop("Description-Content-Type", None)
                     )
 
+                    # Metadata keys that are arrays
+
+                    if importlib_metadata.get("Classifier", None):
+                        for classifier in importlib_metadata["Classifier"]:
+                            python_package_classifier, _ = PythonPackageClassifier.get_or_create(
+                                self._session,
+                                classifier=classifier,
+                            )
+                            HasClassifier.get_or_create(
+                                self._session,
+                                python_package_classifier_id=python_package_classifier.id,
+                                python_package_metadata_id=package_metadata.id,
+                            )
+                        importlib_metadata.pop("Classifier", None)
```

Comment:
WDYT about this way @fridex ?

Reply:
+1, these items have to be removed from the dictionary due to warning produced later on. The code could be simplified to:

```python
                        for classifier in importlib_metadata.pop("Classifier", []):
                            python_package_classifier, _ = PythonPackageClassifier.get_or_create(
                                self._session,
                                classifier=classifier,
                            )
                            HasClassifier.get_or_create(
                                self._session,
                                python_package_classifier_id=python_package_classifier.id,
                                python_package_metadata_id=package_metadata.id,
                            )
```